tests/integration/test_base.py

```diff
@@ -122,3 +122,14 @@ def wait_until_has_class(class_name, elem):
         wait = WebDriverWait(elem, 2)
         wait.until(lambda e: class_name in e.get_attribute('class').split(),
                    u"Class name {} not in {}".format(class_name, elem.get_attribute('class')))
+
+    def wait_for_ajax(self, timeout=15):
```

Comment:
@cahrens In the strictest technical sense, the selenium tests are actually using bok choy, but they're not using the bok choy PageObject API. I don't honestly know/remember the reason we're not using bok choy page objects, but it's probably simply that it we didn't realize it was an option and consider it fully.

I know that native selenium element selectors can be filtered and refined but you can't do that with bok choy (or at least not historically) - i.e. you can't go `self.q(css="".xblock).q(css=".problem-bank").q(css=".item")` which I believe is a pattern we use in these tests.